packages/framework/presence/src/latestValueManager.ts

```diff
@@ -102,7 +102,10 @@ class LatestValueManagerImpl<T, Key extends string>
 		this.value.rev += 1;
 		this.value.timestamp = Date.now();
 		this.value.value = value;
-		this.datastore.localUpdate(this.key, this.value, /* forceUpdate */ false);
+		this.datastore.localUpdate(this.key, this.value, {
+			forceBroadcast: false,
```

Comment:
Should this be set to the controls.force value?

Reply:
There is no controls.force - we can remove froceBroadcast in favor of using allowableUpdateLatency from controls.